configs/config-ccm-psk-tls1_2.h

```diff
@@ -49,6 +49,7 @@
 #define MBEDTLS_MD_C
 #define MBEDTLS_NET_C
 #define MBEDTLS_SHA256_C
+#define MBEDTLS_SHA224_C
```

Comment:
I thought that every config which has 256 must have 224. At least that's the condition implemented in the check-config header file.

Reply:
Ah, good point. Please add a comment stating that SHA-224 is only enabled because the code base doesn't support enabling SHA-256 without SHA-224, and a future version will remove SHA-224.

Reply:
Ok, will do :)

Reply:
Yes, please add it to all the configs that now have an explicit `MBEDTLS_SHA224_C`.